Tools/CISupport/ews-build/steps.py

```diff
@@ -4570,7 +4570,8 @@ def __init__(self, **kwargs):
 
     def start(self, BufferLogObserverClass=logobserver.BufferLogObserver):
         head_ref = self.getProperty('github.base.ref', 'main')
-        self.command = ['git', 'push', 'origin', f'HEAD:{head_ref}']  # FIXME: Support secret remotes
+        remote = self.getProperty('remote', '?')
+        self.command = ['git', 'push', remote, f'HEAD:{head_ref}']  # FIXME: Support secret remotes
```

Comment:
what will happen to this command execution when remote is ?

Reply:
The `git push` command will fail because the remote doesn't exist.